### ISSUE TITLE
Working example?

### ISSUE THREAD

**Issue by 18steps** [rubenCodeforges/angular2-google-api#2]
Looks really great. Would you be able to provide a simple working example?

FWIW, I'm using the Reporting API,
````
const DISCOVERY_DOCS: string[] = [
  'https://analyticsreporting.googleapis.com/$discovery/rest?version=v4',
];
const SCOPE: string = [
  'https://www.googleapis.com/auth/analytics',
  'https://www.googleapis.com/auth/analytics.readonly',
].join(' ');
````

They have a [simple quickstart example here](https://developers.google.com/analytics/devguides/reporting/core/v4/quickstart/web-js), but any API use would be good.

**Comment by rubenCodeforges** [comment]
Ive a test project that uses it try it https://github.com/rubenCodeforges/angular-google-tasks-test-waes/tree/master/src/app if you need something more specific , let me know

**Comment by rubenCodeforges** [comment]
to be more specific the UserService might give you a better look https://github.com/rubenCodeforges/angular-google-tasks-test-waes/blob/master/src/app/common/user/UserService.ts

**Comment by rubenCodeforges** [comment]
@18steps Working on an example module , using gapi Reports will finish it in an hour

**Comment by rubenCodeforges** [comment]
@18steps Ive made an example module , you can add it in your project , make sure to import it in your appModule

The example is located [here](https://github.com/rubenCodeforges/angular2-google-api/tree/master/examples/gapiReporting)

**Comment by 18steps** [comment]
Thanks. I wasn't able to use it directly, one of the reasons is Angular/TypeScript won't do function calls in the module declarations anymore, but I could incorporate a lot of your work and it's been a great help. Instead of initializing configs through the constructors I just import a static config directly into the service.

**Comment by rubenCodeforges** [comment]
@18steps what do you mean `Angular/TypeScript won't do function calls in the module declarations` ?
In the example ?
Ive used angular 4 latest version , it works perfectly.

**Comment by 18steps** [comment]
I wanted to also be able to sign out, because we work with multiple accounts, so I had to incorporate your code and modify it rather than import the package. I ran into an error like this: [Error encountered resolving symbol values statically. Function calls are not supported. Consider replacing the function or lambda with a reference to an exported function](https://github.com/angular/angular-cli/issues/5852)

This was with the newest Angular CLI which uses Webpack
